utils.py

```diff
@@ -100,17 +100,27 @@ def downloadSourceTarball():
              descriptionDone="fetching source tarball...done",
              haltOnFailure=True,
              command=["bash", "-xc", util.Interpolate("""
-  d=/mnt/packages/
-  f="%(prop:tarbuildnum)s_%(prop:mariadb_version)s.tar.gz"
-  find $d -type f -mtime +2 -delete -ls
-  for i in `seq 1 10`;
-  do
-    if flock "$d$f" wget -cO "$d$f" "https://ci.mariadb.org/%(prop:tarbuildnum)s/%(prop:mariadb_version)s.tar.gz"; then
+    d=/mnt/packages/
+    f="%(prop:tarbuildnum)s_%(prop:mariadb_version)s.tar.gz"
+    find $d -type f -mtime +2 -delete -ls
+    res=1
+    for i in {1..10}; do
+      os=$(uname -s)
+      cmd=""
+      if [[ $os != "AIX" ]]; then
+        cmd="flock \"$d$f\" "
+      fi
+      cmd+="wget -cO \"$d$f\" \"https://ci.mariadb.org/%(prop:tarbuildnum)s/%(prop:mariadb_version)s.tar.gz\""
+      if eval $cmd; then
```

Comment:
```diff
-       if eval $cmd; then
+       if eval "$cmd"; then
```

Reply:
Also, for readability, it may be better to use 2 different vars:
```bash
      if [[ $os != "AIX" ]]; then
        use_flock="flock \"$d$f\""
      fi
      cmd="$use_flock wget -cO \"$d$f\" \"https://ci.mariadb.org/%(prop:tarbuildnum)s/%(prop:mariadb_version)s.tar.gz\""
```

Reply:
Done